Source/WebCore/dom/Document.h

```diff
@@ -34,16 +34,11 @@
 #include "DocumentClasses.h"
 #include "DocumentEnums.h"
 #include "DocumentEventTiming.h"
-#include "DocumentSyncData.h"
 #include "FontSelectorClient.h"
-#include "FragmentDirective.h"
 #include "FrameDestructionObserver.h"
 #include "FrameIdentifier.h"
 #include "HitTestSource.h"
-#include "IntDegrees.h"
```

Comment:
Should this be included in Document.cpp to make sure the `using IntDegrees = int32_t;` never goes out of alignment with what's defined in the header?

Reply:
Sure, we can do that.